tests/appsec/iast/test_security_controls.py

```diff
@@ -90,6 +90,7 @@ def setup_no_vulnerability_suppression_with_a_sanitizer_configured_for_a_differe
         self.setup_iast_is_enabled()
         self.r = weblog.post("/iast/sc/s/not-configured", data={"param": "param"})
 
+    @flaky(library="nodejs", reason="APPSEC-58724")
```

Comment:
Since it is flaky just for fastify, the condition can be more precise

```suggestion
    @flaky(library="nodejs" and context.weblog_variant == "fastify", reason="APPSEC-58724")
```